### PR TITLE
Add remaining `border-radius` CSS compat data

### PR DESCRIPTION
**Pull request by ddbeck**
Let me know if you want to see any changes! Thanks!

### CODE REVIEW

File: css/properties/border-bottom-left-radius.json [hunk 1]
```diff
@@ -0,0 +1,158 @@
...
+              },
+              {
+                "prefix": "-webkit-",
+                "version_added": true
+              }
+            ],
+            "firefox": [
+              {
+                "version_added": "4",
+                "notes": "Prior to Firefox 50.0, border styles of rounded corners were always rendered as if <code>border-style</code> was solid. This has been fixed in Firefox 50.0."
```
Comment (Elchi3) [inline css/properties/border-bottom-left-radius.json]: Nit: 50.0 -> 50 (here and below)

File: css/properties/border-bottom-right-radius.json [hunk 2]
```diff
@@ -0,0 +1,158 @@
...
+              },
+              {
+                "prefix": "-webkit-",
+                "version_added": true
+              }
+            ],
+            "firefox": [
+              {
+                "version_added": "4",
+                "notes": "Prior to Firefox 50.0, border styles of rounded corners were always rendered as if <code>border-style</code> was solid. This has been fixed in Firefox 50.0."
```
Comment (Elchi3) [inline css/properties/border-bottom-right-radius.json]: Nit: 50.0 -> 50

File: css/properties/border-top-left-radius.json [hunk 3]
```diff
@@ -0,0 +1,158 @@
...
+              },
+              {
+                "prefix": "-webkit-",
+                "version_added": true
+              }
+            ],
+            "firefox": [
+              {
+                "version_added": "4",
+                "notes": "Prior to Firefox 50.0, border styles of rounded corners were always rendered as if <code>border-style</code> was solid. This has been fixed in Firefox 50.0."
```
Comment (Elchi3) [inline css/properties/border-top-left-radius.json]: Nit: 50.0 -> 50

**Comment by ddbeck** [comment]
Fixed the extraneous ".0" version numbers! Thanks, @Elchi3!

**Pull request merged**